corehq/apps/analytics/tasks.py

```diff
@@ -38,6 +38,19 @@
 HUBSPOT_EXISTING_USER_INVITE_FORM = "7533717e-3095-4072-85ff-96b139bcb147"
 HUBSPOT_COOKIE = 'hubspotutk'
 
+ANALYTICS_RETRIES = 3
+ANALYTICS_SLEEP = 5
+
+def _persistent_analytics_post(func, retries=ANALYTICS_RETRIES, sleep=ANALYTICS_SLEEP):
+    for i in range(retries):
+        try:
+            return func()
+        except (requests.exceptions.HTTPError, requests.exceptions.BaseHTTPError):
+            if i < retries - 1:
+                time.sleep(sleep)
```

Comment:
If this is only called via celery tasks is it not worth having celery handle the retry e.g. https://github.com/dimagi/commcare-hq/blob/4ece5b08a806f13ba37e12e60adc289cc9604334/corehq/apps/hqwebapp/tasks.py#L12-L12